integration-cli/docker_api_containers_test.go

```diff
@@ -432,7 +432,7 @@ func (s *DockerSuite) TestGetStoppedContainerStats(c *check.C) {
 	}()
 
 	// allow some time to send request and let daemon deal with it
-	time.Sleep(1 * time.Second)
+	time.Sleep(time.Second)
```

Comment:
@LK4D4 yeah it's just to make all *sleep 1 second* looks the same. ``time.Sleep(1 * time.Second)`` or ``time.Sleep(time.Second)`` I don't really have a strong opinion :sweat_smile:. 

Maybe I'm a little OCD on that though :sweat: but as I had all the ``time.Sleep`` in integration-cli in front of me, so it caught my eyes :blush:.